AzUtilities/AzBlob.py

```diff
@@ -0,0 +1,265 @@
+"""
+Class for file-like communication with Azure Blobstorage
+
+requirements
+azure==2.0.0
+azure-batch==3.0.0
+azure-common==1.1.7
+azure-servicefabric==5.6.130
+azure-servicemanagement-legacy==0.20.6
+azure-storage==0.34.3
+tempfile==
+warnings
+"""
+
+import tempfile as tmp
+from azure.storage.blob import BlockBlobService
+from azure.storage.blob import AppendBlobService
+import os
+import io
+import warnings
+
+
+class AzBlob:
+    """Class to communicate with Azure Blob storage as file like object. It works with block and append blobs
+
+    The idea is to work with temporary files in the the local temp folder and upload or download them
+    as needed. The temporary file will be deleted each time open(), close() or delete_temp() are called"""
+
+    def __init__(self, account_name, account_key):
+        """
+
+        :param account_name: name of the blob account
+        :param account_key: key of the blob account
+        """
+        self.block_blob_service = BlockBlobService(account_name=account_name, account_key=account_key)
+        self.append_blob_service = AppendBlobService(account_name=account_name, account_key=account_key)
+        self.name = None
+        self.mode = None
+        self.f = None
+        self.container = None
+        self.blob = None
```

Comment:
Wouldn't it be better if this doesn't exist before calling `open`? There is no other method that accesses it a in the class anyway. Similar to some variables above, e.g. `mode`